examples/CIFAR10_with_Eigenvalue_Decay.py

```diff
@@ -0,0 +1,148 @@
+from __future__ import print_function
```

Comment:
I agree, the user only need to know examples of the syntax:

from keras.regularizers import EigenvalueRegularizer
...
model.add(Dense(512, W_regularizer=EigenvalueRegularizer(0.1)))